README.md

```diff
@@ -44,7 +46,9 @@ The project aims to:
     * [Docker Setup for Servers and Local Machines](./docs/setup-docker.md)
         * Nvidia is recommended for now, but support for other GPUs should be development.
         * Local use is not recommended.
+
 ## Resources
+These are a few links to documentation and guides that have proven useful during development.
```

Comment:
I dont think that's why I put those there. I put those there cause that's what we use.